src/components/Card.js

```diff
@@ -11,10 +11,11 @@ const Card = ({ handleScore, id}) => {
     <div onClick={handleClick}>
       <img 
         src={cardImages[id]}
+        className='card'
         style={{
-          width: '200px'
+          
         }}
-        alt='buttholes'
+        alt={id}
```

Comment:
Hmmm this is interesting! Thinking about accessibility for the game. The `id` is just a number, right? Would the intention be that someone was relying on the alt text would play the game based on the number, rather than the image?

Reply:
yyeeeeeep